### PR TITLE
Improve Morse code support

### PR DESCRIPTION
**Pull request by colinvh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ five.upHigh() // ⁵
 five.downLow() // ₅
 five.tooSlow() // 5, with a ~500 millisecond delay
 five.roman() // V
-five.morseCode() // di-di-di-di-dit
+five.morseCode() // di-di-di-di-dah
```
Comment (jackdcrawford) [inline README.md]: `// dot-dot-dot-dot-dash`?

**Comment by jackdcrawford** [comment]
Minor comment, happy to merge after! Thanks :neckbeard: :+1:

File: README.md (hunk 1 above)
Comment (colinvh) [inline README.md]: The generally accepted way to pronounce Morse is with "dit"s and "dah"s, and a non-final "dit" is shortened to "di-".  http://www.acsu.buffalo.edu/~maxwell/Fundamental_Morse.html

Comment (jackdcrawford) [inline README.md]: Great, surprising how much five can teach you! Thanks @colinvh

**Pull request merged**